src/front-end/typescript/lib/pages/opportunity/list.tsx

```diff
@@ -364,7 +364,22 @@ const Header: ComponentView<State, Msg> = () => {
           </Link>
         </div>
       </Col>
-      <Col xs='12' md='6'>
+
+      <Col xs='12' md='4' className='mb-4 mb-md-0'>
+        <div className='rounded bg-c-opportunity-list-learn-more-bg p-4 h-100 d-flex flex-column align-items-start flex-nowrap'>
+          <ProgramType type_='new' className='mb-2' />
+          <p className='mb-3 font-size-small'><em>New Product</em> opportunities are still being scoped out.</p>
+          <Link
+            className='font-size-small mt-auto'
+            symbol_={rightPlacement(iconLinkSymbol('arrow-right'))}
+            iconSymbolSize={0.9}
+            dest={routeDest(adt('learnMoreSWU', null))}>
```

Comment:
This is currently linking to the SWU page; updating the link is a separate ticket